CMakeLists.txt

```diff
@@ -161,7 +162,11 @@ else()
 endif()
 
 # Build Tests and download Catch2
-if (${ENABLE_UNIT_TESTS} OR ${ENABLE_INTEGRATION_TESTS} OR ${ENABLE_REGRESSION_TESTS})
+if (${ENABLE_UNIT_TESTS} OR ${ENABLE_INTEGRATION_TESTS} OR ${ENABLE_REGRESSION_TESTS} OR ${ENABLE_PERFORMANCE_TESTS})
+
+  if (${ENABLE_PERFORMANCE_TESTS})
+    add_definitions(-DCATCH_CONFIG_ENABLE_BENCHMARKING)
```

Comment:
Instead of this, you should hang this off of `catch2_define` in `tst/CMakeLists.txt`:
```cmake
if (ENABLE_PERFORMANCE_TESTS)
    target_compile_definitions(catch2_define PUBLIC CATCH_CONFIG_ENABLE_BENCHMARKING)
endif()
```

Reply:
Done.